compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1928,10 +1928,107 @@ OMR::ARM64::TreeEvaluator::BNDCHKEvaluator(TR::Node *node, TR::CodeGenerator *cg
    return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
    }
 
+static TR::Instruction *compareIntsAndBranchForBNDCHK(TR::ARM64ConditionCode  branchType,
+                                           TR::Node             *node,
+                                           TR::CodeGenerator    *cg,
+                                           TR::SymbolReference  *sr)
+	{
+	TR::Node *secondChild = node->getSecondChild();
+	TR::Node *firstChild = node->getFirstChild();
+	TR::LabelSymbol *snippetLabel = generateLabelSymbol(cg);
+	TR::Register *src1Reg = cg->evaluate(firstChild);
```

Comment:
Is there any reason for placing `firstChild` after `secondChild`?
Please consider changing the order of these lines.